cli/init.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cli
+
+import (
+	"errors"
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+const initLongDesc = `
+This command can be used to initialize a new template.
+`
+
+type initCmd struct {
+	out io.Writer
+}
+
+func newInitCmd(out io.Writer) *cobra.Command {
```

Comment:
>newInitCmd [](start = 5, length = 10)

will init be a good place to preload the `system` build step container images?

Reply:
yeah, we could do that :+1: - made a work item to discuss more about that